spec/controllers/api/v1/authenticated_user.rb

```diff
@@ -0,0 +1,7 @@
+shared_context 'Authenticated User' do
+  let(:user) { create(:user) }
+
+  before do
+    request.headers.merge! user.create_new_auth_token
```

Comment:
As i say above, make a method in a helper which authenticate user on CALL not always